docs/getting_started/installation.md

```diff
@@ -1,90 +1,49 @@
-conda config --append pkgs_dirs /ofo-share/repos-david/conda/pkgs/
+conda create -n geograypher python=3.9 -y
+conda activate geograypher
 ```
 
-Now you should see all of my conda environments when you do `conda env list` . The one you want is `geograypher-stable` , and can be activated as follows:
-
+Install Geograypher:
 ```
-conda activate geograypher-stable
+pip install geograypher
 ```
 
-Use this instead of `geograypher` in future steps.
-
-## Creating a new environment
-
-> For internal collaborators working on `/ofo-share`, you can opt to store the new environment on `/ofo-share` so that you can access it from any VM. First make sure the location where you want to store the env is set to the default for conda:
->
-> ```
-> conda config --prepend envs_dirs /ofo-share/repos-<yourname>/conda/envs/
-> conda config --prepend pkgs_dirs /ofo-share/repos-<yourname>/conda/pkgs/
-> ```
->
-> You will need to run the above two lines on all new VMs before you can activate the env.
-
-Create and activate the environment:
+## Advanced/Developer Installation
+Create and activate a conda environment:
 
 ```
 conda create -n geograypher python=3.9 -y
 conda activate geograypher
 ```
 
-> For internal collaborators working on /ofo-share, you could run into permissions issues when installing dependencies. Check that your executable permissions are valid by running python and python3.9.
->
-> ```
-> python
-> python3.9
-> ```
->
-> If you get a permission denied error, get the location of the python executable inside of your conda environment (from the error message).
->
-> Use the output and change the permissions using chmod. What the command should look like:
->
-> ```
-> chmod ugo+x <CONDA ENV LOCATION>/bin/python3.9
-> ```
-
-If you haven't already, install [poetry](https://python-poetry.org/docs/):
+Install [poetry](https://python-poetry.org/docs/):
 
 ```
 curl -sSL https://install.python-poetry.org | python3 -
 ```
 
-Now use this to install the majority of dependencies. First `cd` to the directory containing the `geograypher` repo. Then run:
+Now use this to install the majority of dependencies. First cd to the directory containing the `geograypher` repo. Then run:
 
 ```
 poetry install
 ```
 
-Now install the `pytorch3d` dependencies that can't be installed with `poetry`:
-
-```
-conda install pytorch=1.13.0 torchvision pytorch-cuda=11.6 -c pytorch -c nvidia -y
-conda install -c fvcore -c iopath -c conda-forge fvcore iopath -y
-conda install -c bottler nvidiacub -y
-conda install pytorch3d -c pytorch3d -y
-```
-
-Validate the installation
-
-```
-python -c "import torch; print(torch.cuda.is_available())"
-python -c "import pytorch3d; print(pytorch3d.__version__)"
-```
-
 If you are working on a headless machine, such as a remote server, you will need the [XVFB](https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml) package to provide a virtual frame buffer. This can be installed at the system level using the package manager, for example:
```

Comment:
Can you actually make a separate section for this before the Pytorch3D one? XVFB would need to be installed even if you use the pip install if you are on a headless machine.

Reply:
Should this information go in the README also then?

Reply:
I'll leave that to you. The number of people working on headless machines should be pretty low so I'm not sure it's worth including it.